features/other_b/series_lock.feature

```diff
@@ -4,84 +4,60 @@ Feature: Locked and partially locked series
   As a registered archive user
   I should be able to make my serial works visible only to other registered users
 
-Scenario: Post a series with a restricted work, then add a draft, then make the draft public and post it
-    Given I am logged in as "fandomer" with password "password"
-      And basic tags
-      And I go to the new work page
-      And I select "Not Rated" from "Rating"
-      And I check "No Archive Warnings Apply"
-      And I fill in "Fandoms" with "Supernatural"
-      And I fill in "Characters" with "Sammy"
-      And I fill in "Work Title" with "Humbug"
-      And I fill in "content" with "The story of how they met and how they got into trouble"
+  Scenario: Post a series with a restricted work, then add a draft, then make the draft public and post it
+    Given I am logged in as "fandomer"
+      And I post the locked work "Humbug" as part of a series "Antiholidays"
+    Then I should see "Part 1 of the Antiholidays series"
+    When I go to fandomer's series page
+    Then I should see "Antiholidays"
+    When I am logged out
+     And I go to fandomer's series page
+    Then I should not see "Antiholidays"
+    When I am logged in as "reccer"
+      And I go to fandomer's series page
+    Then I should see "Antiholidays"
+    When I view the series "Antiholidays"
+    Then I should see "Humbug"
+    When I am logged out
```

Comment:
Instead of `When I am logged out/And I am logged in as "fandomer"`, we can just do `When I am logged in as "fandomer"` -- the log in step takes care of logging out, too. :D